server/pulp/server/webservices/http.py

```diff
@@ -210,7 +154,7 @@ def ssl_client_cert():
     @rtype: str or None
     @return: pem encoded cert
     """
-    return _get_wsgi_environ().get('SSL_CLIENT_CERT', None)
+    return _thread_local.wsgi_environ.get('SSL_CLIENT_CERT', None)
```

Comment:
This won't work the way it is written because the line that set wsgi_environ in thread locals is now gone. It was here: https://github.com/pulp/pulp/pull/1830/files#diff-9a48cb728a264356eaf4f26c0f2286e5L68

Look at the code that is calling this and determine if everything can be gotten from the django request object.